docs/model/directory.rst

```diff
@@ -0,0 +1,5 @@
+Push items: directory
+================
```

Comment:
I think this will generate a warning as the '=' below the heading doesn't have the right number of characters?

```suggestion
Push items: directory
=====================
```